src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -1009,6 +1009,11 @@ internal NamedTypeSymbol GetUnboundGenericTypeOrSelf()
         /// </summary>
         internal abstract bool HasCodeAnalysisEmbeddedAttribute { get; }
 
+        /// <summary>
+        /// Gets a value indicating whether this type has System.Runtime.CompilerServices.InterpolatedStringHandlerAttribute or not.
+        /// </summary>
+        internal abstract bool IsInterpolatedStringHandlerType { get; }
```

Comment:
Could this be `virtual` and just return `false` by default?

Reply:
As a general design principle, we don't use virtual methods in the base symbol types. Too easy to miss an implementation and introduce a bug.